packages/database/src/migrations/20210714013524-DeleteDuplicateCountriesFromEntityTable-modifies-data.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+import { arrayToDbString } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const duplicateCountryEntityIds = [
+  "'60da867061f76a20cf01e23f'", // duplicate Fiji
+  "'60eb91df61f76a7f770790a3'", // duplicate Papua New Guinea
+  "'60949c6061f76a622c020231'", // duplicate Samoa
+];
+
+const realCountryEntityIds = [
+  "'5d51f501f013d6320f3cf633'", // real Fiji
+  "'5d3f8844f327a531bfd8ad77'", // real Papua New Guinea
+  "'5df1b88c61f76a485cd1ca09'", // real Samoa
+];
```

Comment:
If you store an object mapping the country code to the id, you can then refer to that in all the places you currently have an id directly in the code (which is a no-no, because it's so much easier to get an id wrong or not know what country an id relates to compared with a named variable or using the code).

```suggestion
const duplicateEntityIdByCountry = {
  FJ: "'60da867061f76a20cf01e23f'", // duplicate Fiji
  PG: "'60eb91df61f76a7f770790a3'", // duplicate Papua New Guinea
  WS: "'60949c6061f76a622c020231'", // duplicate Samoa
};

const realEntityIdByCountry = {
  FJ: "'5d51f501f013d6320f3cf633'", // real Fiji
  PG: "'5d3f8844f327a531bfd8ad77'", // real Papua New Guinea
  WS: "'5df1b88c61f76a485cd1ca09'", // real Samoa
};
```

You can still get the arrays you have currently whereever you need them, using `Object.values`

```
const duplicateCountryEntityIds = Object.values(duplicateEntityIdByCountry);
const realCountryEntityIds = Object.values(realEntityIdByCountry);
```